bin/rspecq

```diff
@@ -3,6 +3,7 @@ require "optparse"
 require "rspecq"
 
 DEFAULT_REDIS_HOST = "127.0.0.1"
+DEFAULT_REDIS_URL = "redis://127.0.0.1:6379/0"
```

Comment:
Can we get away without specifying a default here? redis-rb doesn't, so we could do the same, for the shake of parity and simplicity. (Well redis-rb defaults to `ENV["REDIS_URL"]` but in our case this is not desirable).

Reply:
Yes, I only added this to match the host convention. I'll Remove it.

Reply:
```suggestion
```